pahfit/base.py

```diff
@@ -463,6 +469,19 @@ def save(self, obs_fit, filename, outform):
                     ]
                 )
             elif comp_type == "Drude1D":
+
+                # Calculate feature strength.
+                # TODO: Use input units to determine conversion and feature strength units.
```

Comment:
Note TODO in case of future changes to units to which input spectrum is converted.

Reply:
We should probably abstract away all unit conversion (input -> model -> output) from the main fitting code itself.  I think we decided to standardize on f_nu units, and their surface brightness equivalents (per sr), but otherwise any combination should be allowed.  I'll mention in an issue.